### PR TITLE
(MAINT) Drop MockVsphereHelper

### PR DESCRIPTION
**Pull request by Dakta**
No longer relevant since vmpooler replaced vcloudpooled.

### CODE REVIEW

**Comment by Dakta** [comment]
Jenkins, retest this please.

File: Gemfile [hunk 1]
```diff
@@ -19,7 +19,7 @@ end
 # don't want to create a transitive dependency
 group :acceptance_testing do
   gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.0')
-  gem "beaker-abs", *location_for(ENV['ABS_VERSION'] || '~> 0.3.0')
+  # gem "beaker-abs", *location_for(ENV['ABS_VERSION'] || '~> 0.3.0')
```
Comment (kevpl) [inline Gemfile]: please delete this line rather than comment it out

File: spec/beaker/hypervisor/vmpooler_spec.rb [hunk 2]
```diff
@@ -147,20 +143,15 @@ module Beaker
...
-        vmpooler.cleanup
+        vm_count = vmpooler.instance_variable_get( :@hosts ).count
 
-        hosts = vmpooler.instance_variable_get( :@hosts )
-        hosts.each do | host |
-          name = host.name
-          vm = MockVsphereHelper.find_vm( name )
-          expect( vm.runtime.powerState ).to be === "poweredOn" #handed back to the pool, stays on
-        end
+        expect( Net::HTTP ).to receive( :new ).exactly(vm_count).times.and_return( mock_http )
```
Comment (kevpl) [inline spec/beaker/hypervisor/vmpooler_spec.rb]: please pick a spacing style for method parameters & apply it consistently across your changes.

File: Gemfile (hunk 1 above)
Comment (kevpl) [reply Gemfile]: still needs deletion..

**Pull request merged**